File: packages/android/src/main/java/com/kozmos/components/POIResultList/POIResultList.figma.kt
package com.kozmos.components.poiresultlist

import androidx.compose.runtime.Composable
import com.figma.code.connect.Figma
import com.figma.code.connect.FigmaConnect
import com.figma.code.connect.FigmaProperty
import com.figma.code.connect.FigmaType

@FigmaConnect("https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1351-8171")
class KozmosPOIResultListConnect {
    @FigmaProperty(FigmaType.Text, "Result Count Text")
    val resultCountLabel: String = "12 results"

    @Composable
    fun ComponentExample() {
        KozmosPOIResultList(
            items = emptyList(),
            resultCountLabel = resultCountLabel,
            onSelect = {}
        )
    }
}

File: packages/ios/Sources/Components/POIResultList/POIResultList.figma.swift
import SwiftUI
import Figma

struct KozmosPOIResultListConnect: FigmaConnect {
    let component = KozmosPOIResultList<EmptyView>.self
    let figmaNodeUrl = "https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=1351-8171"

    @FigmaString("Result Count Text")
    var resultCountLabel: String = "12 results"

    var body: some View {
        KozmosPOIResultList(
            items: [],
            resultCountLabel: self.resultCountLabel,
            onSelect: { _ in }
        ) {
            EmptyView()
        }
    }
}
